### PR TITLE
Refactoring a too big function

### PR DESCRIPTION
**Pull request by Vane11ope**
now is good time to clean up the messy functions

### CODE REVIEW

**Comment by nsajko** [comment]
I think @radare prefers for commit messages to start with a capital letter.

File: libr/core/panels.c [hunk 1]
```diff
@@ -213,105 +215,106 @@ static void panelPrint(RCore *core, RConsCanvas *can, RPanel *panel, int color)
...
+		r_cons_canvas_box (can, panel->pos.x, panel->pos.y, w, h, core->cons->pal.graph_box2);
+	} else {
+		r_cons_canvas_box (can, panel->pos.x, panel->pos.y, w, h, core->cons->pal.graph_box);
+	}
+}
+
+static void menuPanelPrint(RConsCanvas *can, RPanel *panel, int x, int y, int w, int h) {
+	char *text;
+	(void) r_cons_canvas_gotoxy (can, panel->pos.x + 2, panel->pos.y + 2);
+	text = r_str_ansi_crop (panel->title, x, y, w, h);
```
Comment (radare) [inline libr/core/panels.c]: Define and assign in the same line

File: libr/core/panels.c [hunk 2]
```diff
@@ -213,105 +215,106 @@ static void panelPrint(RCore *core, RConsCanvas *can, RPanel *panel, int color)
...
+	} else {
+		cmdStr = r_core_cmd_str (core, panel->cmd);
+	}
+	if (y < 0) {
+		y = 0;
+	}
+	if (x < 0) {
+		char *white = (char*)r_str_pad (' ', 128);
+		int idx = -x;
+		if (idx >= sizeof (white)) {
```
Comment (radare) [inline libr/core/panels.c]: Sizeof? This should be strlen and looks like RMIN will do a better job here

File: libr/core/panels.c [hunk 3]
```diff
@@ -213,105 +215,106 @@ static void panelPrint(RCore *core, RConsCanvas *can, RPanel *panel, int color)
...
+			idx = sizeof (white) - 1;
+		}
+		white[idx] = 0;
+		text = r_str_ansi_crop (cmdStr,
+				0, y + graph_pad, w + x - 3, h - 2 + y);
+		char *newText = r_str_prefix_all (text, white);
```
Comment (radare) [inline libr/core/panels.c]: Isnt newText leaking?

**Comment by Vane11ope** [comment]
done

**Pull request merged**